test/Yoti.Auth.Tests/ActivityTests.cs

```diff
@@ -125,11 +125,11 @@ public void Activity_AddAttributesToProfile_FullName()
         [TestMethod]
         public void Activity_AddAttributesToProfile_GivenNames()
         {
-            string value = "GivenNamesValue";
+            const string value = "GivenNamesValue";
 
             var attribute = new AttrpubapiV1.Attribute
             {
-                Name = _givenNamesAttribute,
+                Name = GivenNamesAttribute,
```

Comment:
We don't need to define a different test value for every test.  We only care that it's a String.  We set the String value, then prove that we get it back when we ask for it.

We can just define _"someStringValue"_ or similar, then reuse it.

Reply:
Done 👍